lib/WP_Auth0_Lock10_Options.php

```diff
@@ -134,13 +134,8 @@ protected function build_settings( $settings ) {
 
 	public function get_sso_options() {
 		$options['scope'] = WP_Auth0_LoginManager::get_userinfo_scope( 'sso' );
-
```

Comment:
This is only ever used for SSO, which uses the hybrid auth (modified implicit) so there's never a need for a code callback processing.